.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I didn't realise we had it in the package.json, I'll remove the file. Good spot!